misk/src/main/kotlin/misk/web/interceptors/ConcurrencyLimitsInterceptor.kt

```diff
@@ -172,7 +173,8 @@ internal class ConcurrencyLimitsInterceptor internal constructor(
         action = action,
         defaultLimiter = createLimiterForAction(action, quotaPath = null),
         clock = clock,
-        logLevel = config.concurrency_limiter_log_level
+        logLevel = config.concurrency_limiter?.log_level
```

Comment:
the `concurrency_limiter.log_level` will default to `config.concurrency_limiter_log_level` if `concurrency_limiter` isn't specified